src/Common/src/System/Net/Http/TlsCertificateExtensions.cs

```diff
@@ -73,6 +73,24 @@ private static bool IsClientCertificate(X509Certificate2 cert)
             return isClientAuth && isDigitalSignature;
         }
 
+        internal static X509Chain BuildNewChain(X509Certificate2 certificate)
+        {
+            var chain = new X509Chain();
+            chain.ChainPolicy.VerificationFlags = X509VerificationFlags.AllFlags;
+            chain.ChainPolicy.RevocationFlag = X509RevocationFlag.ExcludeRoot;
+            chain.ChainPolicy.ApplicationPolicy.Add(s_clientCertOidInst);
```

Comment:
Why was it correct before then for the other use?  Wasn't that also a single-cert case? https://github.com/dotnet/corefx/blob/master/src/Common/src/System/Net/Http/TlsCertificateExtensions.cs#L92-L111